packages/fuels/tests/harness.rs

```diff
@@ -4180,3 +4180,119 @@ async fn test_vector() -> Result<(), Error> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn test_output_variable_estimation() -> Result<(), Error> {
+    abigen!(
+        MyContract,
+        "packages/fuels/tests/test_projects/token_ops/out/debug/token_ops-abi.json"
+    );
+
+    let wallet_config = WalletsConfig::new(Some(3), None, None);
+    let wallets = launch_custom_provider_and_get_wallets(wallet_config, None).await;
+
+    let contract_id = Contract::deploy(
+        "tests/test_projects/token_ops/out/debug/token_ops.bin",
+        &wallets[0],
+        TxParameters::default(),
+        StorageConfiguration::default(),
+    )
+    .await?;
+
+    let contract_instance = MyContract::new(contract_id.to_string(), wallets[0].clone());
+    let contract_methods = contract_instance.methods();
+    let mint_asset_id = AssetId::from(*contract_id.hash());
+    let addresses: [Address; 3] = wallets
```

Comment:
Unrelated note: maybe we should get some methods on `Vec<Wallets>` that provide eg the vec of their addresses, the vec of their balances, etc? Wdyt?

Reply:
Maybe we could implement it as a wrapper for `Vec<Wallets>`. Good idea IMO.

Reply:
Nice, can you open an issue for this?

Reply:
I've added it as a discussion for now so that others can weigh in.